io/plugins/eu.esdihumboldt.hale.io.topojson.test/src/eu/esdihumboldt/hale/io/topojson/test/TopoJsonInstanceWriterTest.groovy

```diff
@@ -109,10 +107,8 @@ class TopoJsonInstanceWriterTest {
 
 	@Test
 	public void testWriteTopoJson() {
-		def aDate = LocalDate.of(2017, 10, 3)
-		def aTimestamp = Instant.parse('2023-04-02T00:00:00Z')
-		// Create a DateTimeFormatter with the desired pattern
-		def formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss'Z'");
+		def aDate = LocalDate.of(2023, 11, 28)
```

Comment:
@stempler I have used this one as valid date.

Reply:
Ah, missed that it was still used in the first instance.